mandatory/3-playing-computer.js

```diff
@@ -40,3 +40,5 @@ for (let i = 0; i < 5; ++i) {
     console.log(e);
   }
 }
+// f1 one is called once when 'i' is 1 , once when 'i' is 3 and once when i is 5.
```

Comment:
I could be wrong in my approach, but I believe f1 gets called twice, 1st when  i is 1, (iteration's output is 9), and 2nd when i is 3. 
f2 is called 3 times (i=0, then i=2, then i=4, outputting 4, 6, and 8 respectively). I think the i counter starts from 0 upto 4
I could be totally wrong here though hehe!

Reply:
Thank you maha  that cleared things up for me.